torch/_functorch/_aot_autograd/dispatch_and_compile_graph.py

```diff
@@ -336,3 +296,99 @@ def aot_dispatch_autograd_graph(
             maybe_subclass_meta is None
         ), "aot_export_module does not support tensor subclass inputs for now."
     return fx_g, saved_updated_joint_inputs, maybe_subclass_meta
+
+
+def _aot_dispatch_function_wrapper(
+    fn: Callable[..., Any],
+    args: list[Tensor],
+    wrappers: Sequence[Callable[..., Any]],
+    *,
+    aot_config: AOTConfig,
+    functionalized_inputs: Optional[Sequence[Any]] = None,
```

Comment:
It's a bit confusing to me that we need to pass in these extra `functionalized_inputs` arguments, **and** we only need them in `aot_dispatch_autograd_graph` (but not `aot_dispatch_base_graph`)